templates/home/index.html.twig

```diff
@@ -2,57 +2,48 @@
 {{ pageName }} {% endblock %} {% block stylesheet %}
 <link rel="stylesheet" href="assets/css/banner.css" />
 {% endblock %} {% block content %}
-<main>
-    <section
-        id="carouselExampleAutoplaying"
-        class="carousel slide"
-        data-bs-ride="carousel">
-        <div class="main-banner">
-            <ul class="carousel-inner">
-                <li class="carousel-item active">
-                    <div class="item item-1">
-                        <div class="header-text">
-                            <a href="mocktails">
-                                <h2>
-                                    Venez!<br />vite découvrir nos mocktails
-                                </h2>
-                            </a>
-                        </div>
+<section
+    id="carouselExampleAutoplaying"
+    class="carousel slide"
+    data-bs-ride="carousel"
+>
+    <div class="main-banner">
+        <ul class="carousel-inner">
+            <li class="carousel-item active">
+                <div class="item item-1">
+                    <div class="header-text">
+                        <a href={{ path('app_mocktail_index') }}>
+                            <h2>Venez!<br />vite découvrir nos mocktails</h2>
```

Comment:
H1 ?

Reply:
but you asked me to change it into h2 last time
![Capture](https://github.com/WildCodeSchool/Rogue-One_Symfony/assets/99322580/cbb32de9-581c-449a-8cca-3aa39ebdbb05)